esmvalcore/preprocessor/_time.py

```diff
@@ -341,6 +341,9 @@ def extract_season(cube: Cube, season: str) -> Cube:
     season:
         Season to extract. Available: DJF, MAM, JJA, SON
         and all sequentially correct combinations: e.g. JJAS
+    full:
```

Comment:
I would add more documentation or maybe context. I know what the issue is, but I think a person without context might get confused.